core/src/main/java/org/fao/geonet/kernel/setting/Settings.java

```diff
@@ -82,7 +82,7 @@
     public static final String SYSTEM_SELECTIONMANAGER_MAXRECORDS = "system/selectionmanager/maxrecords";
     public static final String SYSTEM_CSW_ENABLE = "system/csw/enable";
     public static final String SYSTEM_CSW_ENABLEWHENINDEXING = "system/csw/enabledWhenIndexing";
-    public static final String SYSTEM_CSW_CONTACT_ID = "system/csw/contactId";
+    public static final String SYSTEM_CSW_CAPABILITY_RECORD_ID = "system/csw/capabilityRecordId";
```

Comment:
Could not be better to use the metadata uuid? Should be more stable than internal ids.

Suppose that the user uses the import option to update the metadata, this option afaik removes the old record and creates a new one with a new id.

Reply:
Done.